runtime/gc_glue_java/CompactSchemeFixupObject.cpp

```diff
@@ -87,6 +86,13 @@ MM_CompactSchemeFixupObject::fixupContinuationObject(MM_EnvironmentStandard *env
 	}
 }
 
+void
+MM_CompactSchemeFixupObject::fixupContinuationObject(MM_EnvironmentStandard *env, omrobjectptr_t objectPtr)
+{
+	fixupContinuationNativeSlots(env, objectPtr);
+	fixupMixedObject(objectPtr);
```

Comment:
Changed the order of these operations from the original order. Don't remember it's relevant, but just in case, better to mention it.